api/entity/v1/reqres/bootstrap.go

```diff
@@ -21,10 +22,29 @@ type BootstrapVerifyResponse struct {
 	// Hash of the decrypted plaintext
 	Hash string `json:"hash"`
 	// Error code if operation failed
-	Err errors.ErrorCode `json:"err,omitempty"`
+	Err sdkErrors.ErrorCode `json:"err,omitempty"`
```

Comment:
All errors that the SDK consumes and returns are typed, and they inherit from `./errors/SDKError` -- SDKError has methods to Wrap/Unwrap errors, and also can  convert an `./errors/ErrorCode` into a SDKError (useful for getting an error code from a REST response and map it to a concrete SDKError instance).